viewer/packages/rendering/src/CadMaterialManager.ts

```diff
@@ -145,6 +145,8 @@ export class CadMaterialManager {
       throw new Error(`Model identifier: ${modelIdentifier} not found`);
     }
 
+    texture.encoding = THREE.sRGBEncoding;
```

Comment:
Just to be entirely sure - do we know that this is needed? E.g. has the code been tested with and without it?

Reply:
Texture from GLTF model are stored in `sRGB` colorspace, in threejs texture colorspace is `Linear` by default and `sRBGEncoding` has to be applied without which color value will defer, I can add refer to the details as a comment

Reply:
Okay :+1: I don't think a link is needed here - but you I won't stop you if you want one :P